packages/mask/src/plugins/Game/hook/useGameList.ts

```diff
@@ -1,17 +1,21 @@
 import { useAsync } from 'react-use'
-import { useWeb3Connection } from '@masknet/plugin-infra/web3'
-import { NetworkPluginID } from '@masknet/web3-shared-base'
+import { useWeb3State } from '@masknet/plugin-infra/web3'
 import { useGameConstants } from '@masknet/web3-shared-evm'
 import { EMPTY_LIST } from '@masknet/shared-base'
-import { PluginGameRPC } from '../messages'
-import type { GameInfo } from '../types'
+import type { GameRSSNode } from '../types'
 
 export function useGameList() {
-    const connection = useWeb3Connection(NetworkPluginID.PLUGIN_EVM)
+    const { Storage } = useWeb3State()
```

Comment:
If the currently selected network isn't EVM, then `Storage.createRSSStorage` will throw any error.